src/Fls.Results/Fls.Results.csproj

```diff
@@ -1,7 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-  </PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
+    <PackageId>Fls.Results</PackageId>
+    <Title>Fls.Results</Title>
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Authors>FirstLine Software .Net Core Practice Team</Authors>
+    <Company>FirstLine Software</Company>
+    <license>https://raw.githubusercontent.com/vkamiansky/results/master/LICENSE</license>
+    <PackageProjectUrl>https://github.com/vkamiansky/results</PackageProjectUrl>
+    <PackageTags>c#;result;railroad</PackageTags>
+    <RepositoryUrl>https://github.com/vkamiansky/results.git</RepositoryUrl>
+    <Description>A C# framework for building railroad-oriented code with results.</Description>
+    <Copyright>Copyright (c) 2020, FirstLine Net Core Practice Team</Copyright>
```

Comment:
still abides.

Reply:
I don't understand, what this comment about.